org.lflang.tests/src/org/lflang/tests/TestBase.java

```diff
@@ -430,7 +433,10 @@ private void configure(LFTest test, Configurator configurator, TestLevel level)
             if (!configurator.configure(test)) {
                 throw new TestError("Test configuration unsuccessful.", Result.CONFIG_FAIL);
             }
-            context.loadTargetConfig(); // Reload in case target properties have changed.
+            // FIXME: The line below is commented out because it reloads the target properties
+            //  and thus overwrites changes done by the configurator. I am sure it was there 
+            //  for a reason. Thus the FIXME...
+            // context.loadTargetConfig(); // Reload in case target properties have changed.
```

Comment:
OBS. What to do about this. I don't understand the comment either. Why would we want to reload and overwrite the target properties if they have  changed?

Reply:
I agree that in its current form it makes no sense, but I think the intent here is to "reset" the configuration in between test runs. Hence, if this line is just moved to first line of the block starting with `if (configurator != null)`, I think it would work as intended?